packages/adminrouter/extra/src/test-harness/tests/test_agent.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) Mesosphere, Inc. See LICENSE file for details.
+
+import grp
+
+import psutil
+import pytest
```

Comment:
The issue here is actually something that we should solve a bit more universally. I would love it if these  unittests were triggered with the other code quality tests, but the top-level code quality job will run with different dependencies than production code. See: https://jira.mesosphere.com/browse/DCOS-14305

For now, making flake8 during the AR jenkins jobs seems good to me